### DISCUSSION TITLE
Understanded configuration

### DISCUSSION THREAD

**Discussion by Smaillor** [DerFrZocker/Custom-Ore-Generator#32] [Q&A]
Hi, I don't understand the plugin configuration. Can you help me ?

**Comment by DerFrZocker** [comment]
What don't you understand / what do you want to do? You can find some configurations in other discussions here.

**Reply by Smaillor** [reply]
I don't understand how I can create a custom generation with oraxen

**Reply by DerFrZocker** [reply]
To generated a Oraxen block follow the following steps:

1. Place the custom block
2. Use `/oregen create`
3. Enter a name like `MyCustomOre`
4. Left click the custom block you placed
5. Type `!next`
6. Left click the blocks you want to replace such as `stone` and `deepslate`
7. Type `!next`
8. Type `!next` again
For normal world generation:
9.  Select `Vanilla Minable Generator` (diamond block)
10. Select `Count Range` (iron block)
11. When it should generated in all biomes click `next` (stick)
12. Set a `vein size` (coal) and click `next` (stick)
13. Set a `height range`, `veins per chunk` and `minimum height` (coal) and click `next` (stick)
14. Click on `Oraxen`
15. Click on `Click to set the found Custom Data` (enchantet bock)
16. Click `next` (stick)
17. Click the world in which the ore should generated
18. Click `next` (stick)
19. Click `Click to create the ore-config` (upper green block)

If your Oraxen block does not generated in new chunks, then there might be an error with you configuration. In this case please send the configuration file found in the directory: `plugins\custom-ore-generator\data\ore-config`
